pkg/forklift-api/webhooks/validating-webhook/admitters/secret-admitter.go

```diff
@@ -62,46 +39,50 @@ func (admitter *SecretAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissio
 			},
 		}
 	}
-	createdForProviderType, ok := secret.GetLabels()["createdForProviderType"]
-	// TODO: need to change this, secret without labels should be allowed
```

Comment:
do we need to take care about dealing non-labeled secret? if so (in the future), i would leave the comment.

Reply:
I removed this TODO because it's handled now - secrets without this label won't be processed by the validation webhook

Reply:
see the ObjectSelector in ValidatingWebhookConfiguration